yarn.lock

```diff
@@ -9690,6 +9690,11 @@ rsvp@^4.8.4:
   resolved "https://registry.yarnpkg.com/rsvp/-/rsvp-4.8.5.tgz#c8f155311d167f68f21e168df71ec5b083113734"
   integrity sha512-nfMOlASu9OnRJo1mbEk2cz0D56a1MBNrJ7orjRZQG10XDyuvwksKbuXNp6qa+kbn839HwjwhBzhFmdsaEAfauA==
 
+rudder-sdk-js@^1.0.16:
```

Comment:
What does this do?

Reply:
Huh... this PR was generated by a script, so I didn't catch that. Let me investigate and I'll get back to you.

Reply:
@hoorayimhelping I'm not familiar with this library, and I can't think of any reason why updating the lsp would pull this in. Do you happen to have any insight into this?

Reply:
Relevant github link: https://github.com/rudderlabs/rudder-sdk-js

Reply:
Seems like it was a dependency that was recently introduced into the UI, but that wasn't tracked yet in the lock file.

Reply:
Relevant PR: https://github.com/influxdata/ui/pull/1455

Reply:
Rudderstack was added with this PR: https://github.com/influxdata/ui/pull/1455. That PR should have included this yarn.lock change, but it was missed. It happens, and it's not a big deal.

This PR should fix it: https://github.com/influxdata/ui/pull/1509

If this flux-lsp change is urgent, it's not a huge deal to merge this in, it just clobbers some of the semantic meaning behind these changes.

Reply:
I'll drop the change just to keep things nice and neat.